src/parquet/parquet.h

```diff
@@ -40,6 +40,26 @@
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+//macros to disable padding. Required for Int96
```

Comment:
Sure! We should create a JIRA for specialization of templates for Int96 with padding as well.
The unit tests will capture both cases.